packages/fluxible-router/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fluxible-router",
-  "version": "2.0.0-beta.4.0",
+  "version": "1.5.4",
```

Comment:
@tom76kimo  This won't work as the master branch has breaking changes for 2.x. You would need to branch from the 1.x branch and push your changes there if you want it released in the 1.x.x branch.

Reply:
Ah, you are right. Will fix it.